recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -419,6 +448,7 @@ DEFAULT_CONFIGURATION = "\
 
 # Not available on Yocto, so we use
 # what comes bundled with Chromium.
+# -Duse_system_flac=1
```

Comment:
The root cause is that `pkg-config` is looking for `flac.pc` in the wrong path. Actually, the `pkg-config` used here is the one located in `/usr/bin/` of your build host, so it looks for `flac.pc` in `/usr/lib` of your build host. In cross compilation, however, `pkg-config` should look for `flac.pc` in the sysroot of the target. To achieve this, [a wrapper](https://chromium.googlesource.com/chromium/src/+/master/build/linux/pkg-config-wrapper) was created in Chromium, but all GYP files in `build/linux/unbundle/` are still incorrectly using `/usr/bin/pkg-config`.